srcopsmetrics/create_bot_knowledge.py

```diff
@@ -129,7 +145,28 @@ def get_only_new_entities(old_data: Dict[str, Any], new_data: PaginatedList) ->
     return [x for x in new_data if x.number in only_new_ids]
 
 
-def load_previous_knowledge(project_name: str, repo_path: Path, knowledge_type: str) -> Dict[str, Any]:
+def load_locally(file_path: Path) -> json:
+    """Load knowledge file from local storage."""
+    _LOGGER.info("Loading knowledge locally...")
+    if not file_path.exists() or os.path.getsize(file_path) == 0:
+        return None
+    with open(file_path, "r") as f:
+        data = json.load(f)
+        results = data["results"]
+    return results
+
+
+def load_remotely(file_path: Path) -> json:
+    """Load knowledge file from CEPH storage."""
+    _LOGGER.info("Loading knowledge from CEPH...")
+    ceph_filename = os.path.relpath(file_path).replace("./", "")
+    try:
+        return get_ceph_store().retrieve_document(ceph_filename)["results"]
+    except NotFoundError:
+        _LOGGER.info("Knowledge %s not found on CEPH" % file_path)
+
+
+def load_previous_knowledge(project_name: str, file_path: Path, knowledge_type: str, use_ceph: bool) -> Dict[str, Any]:
```

Comment:
what about `use_ceph: bool` optional and set to False? like in save_knowledge method?